.eslintrc.js

```diff
@@ -296,6 +296,7 @@ module.exports = {
 						'RangeControl',
 						'SearchControl',
 						'TextareaControl',
+						'ToggleControl',
```

Comment:
I assume we're intentionally adding the rules to:
* RN occurrences
* Readme occurrences
* Test occurrences

I wonder if it makes sense to add the prop to README occurrences at least? Right now, we're suggesting code that will lead to ESLint errors.

Reply:
Good point, I missed some places in the README and JSDocs. I'll do a final check for the rest of the components as well.

(Not sure if a typo, but we're not adding the prop to React Native components.)